backend/users/views/user.py

```diff
@@ -40,12 +41,23 @@ def _validate_channel(name):
 
 
 def _validate_password(password, user):
+    if not password:
+        raise ValidationError("Missing password")
     try:
         validate_password(password, user)
     except djangoValidationError as err:
         raise ValidationError(err.error_list)
 
 
+def _validate_username(username):
+    user_model = get_user_model()
+    try:
+        user_model.objects.get(username=username)
+        raise BadRequestError("Username already exists")
+    except user_model.DoesNotExist:
+        pass
```

Comment:
```suggestion
    if user_model.objects.filter(username=username).exists()
        raise BadRequestError("Username already exists")
```

Reply:
Nice, I had forgotten about the `exists`method!